demo/capture_config.yaml

```diff
@@ -0,0 +1,140 @@
+lcr-modules:
+
+    _shared:
+        lcr-modules: "../"
+        lcr-scripts: "../../lcr-scripts/"
+        root_output_dir: "results/"
+        scratch_directory: "scratch/"
+        unmatched_normal_ids:
+            capture--grch37: "TCRBOA7-N-WEX"
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data/{sample_id}.bam"
+        
+        reference_path:
+            hg38: " --chr_prefixed_genome "                           
```

Comment:
I don't think `--chr_prefixed_genome` will be recognized as a path and this needs to be updated. Also, please include a comment to document what the reference_path means in the config